### PR TITLE
aeskeyfind 1.0 (new formula)

### PR DESCRIPTION
**Pull request by pkirkovsky**
aeskeyfind is a forensic tool for locating 128-bit and 256-bit AES keys in a captured memory image

### CODE REVIEW

File: Library/Formula/aeskeyfind.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
+class Aeskeyfind < Formula
+  homepage "https://citp.princeton.edu/research/memory/code/"
+  url "https://citp.princeton.edu/memory-content/src/aeskeyfind-1.0.tar.gz"
+  sha1 "1b7c267be4b5adec54af2adf67a6733e2bb7896a"
+
+  def install
+    system "make"
+    bin.install "aeskeyfind"
```
Comment (mikemcquaid) [inline Library/Formula/aeskeyfind.rb]: Is there a `make install`?

File: Library/Formula/aeskeyfind.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      0xE7, 0x75, 0xFF, 0x4B, 0x11, 0xE9, 0xE5, 0x67, 0xD1, 0xDD, 0x82, 0x21, 0x36, 0x43, 0x2D, 0xAD,
+      0xBF, 0xB1, 0x92, 0x0D, 0x94, 0xA9, 0x26, 0x43, 0x43, 0xD9, 0x9E, 0xBF, 0x55, 0x4D, 0x22, 0x4C,
+      0xB2, 0x38, 0xDD, 0x07, 0xA3, 0xD1, 0x38, 0x60, 0x72, 0x0C, 0xBA, 0x41, 0xD8, 0xB7, 0xAE, 0xED,
+      0x67, 0x06, 0x3C, 0xE0, 0xF3, 0xAF, 0x1A, 0xA3, 0xB0, 0x76, 0x84, 0x1C, 0xB2, 0x75, 0x7D, 0xD0,
+      0x00, 0x4D, 0xA0, 0xD7, 0xA3, 0x9C, 0x98, 0xB7, 0xD1, 0x90, 0x22, 0xF6, 0x98, 0x24, 0xEC, 0xD3,
+      0xFF, 0x22, 0xD0, 0x33, 0x0C, 0x8D, 0xCA, 0x90, 0xBC, 0xFB, 0x4E, 0x8C, 0xD7, 0x7A, 0x52, 0xB4,
+      0xD7, 0x37, 0xF2, 0x63, 0x74, 0xAB, 0x6A, 0xD4, 0xA5, 0x3B, 0x48, 0x22, 0x3A, 0x76, 0x7F, 0xD5,
+      0xC5, 0x54, 0xAF, 0xE6, 0xC9, 0xD9, 0x65, 0x76, 0x75, 0x22, 0x2B, 0xFA, 0x7E, 0x2E, 0xF8, 0xAA,
+      0x57, 0x4E, 0x76, 0xE9, 0xA9, 0x41, 0x81, 0x3C, 0xD9, 0xAE, 0x33, 0xFF
+    ]
```
Comment (mikemcquaid) [inline Library/Formula/aeskeyfind.rb]: Any way this can be shortened?

File: Library/Formula/aeskeyfind.rb (hunk 1 above)
Comment (pkirkovsky) [inline Library/Formula/aeskeyfind.rb]: No `make install` provided in the original makefile, so I used `bin.install` in the formula.

Is this correct?

File: Library/Formula/aeskeyfind.rb (hunk 2 above)
Comment (pkirkovsky) [inline Library/Formula/aeskeyfind.rb]: Done, see b1bf8f3210243268f04ad688acae4ff33e26fab6

File: Library/Formula/aeskeyfind.rb [hunk 3]
```diff
@@ -0,0 +1,60 @@
+class Aeskeyfind < Formula
+  homepage "https://citp.princeton.edu/research/memory/code/"
```
Comment (mikemcquaid) [inline Library/Formula/aeskeyfind.rb]: It looks from the homepage that this is no longer developed.

File: Library/Formula/aeskeyfind.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/aeskeyfind.rb]: Yeh, thanks.

File: Library/Formula/aeskeyfind.rb (hunk 3 above)
Comment (pkirkovsky) [inline Library/Formula/aeskeyfind.rb]: It's a stable purpose-built tool that works well and is included in popular distros:

https://packages.debian.org/search?keywords=aeskeyfind
http://packages.ubuntu.com/search?keywords=aeskeyfind

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**